examples/hostpath-provisioner/go.mod

```diff
@@ -3,8 +3,8 @@ module sigs.k8s.io/kubernetes-sigs/sig-storage-lib-external-provisioner/examples
 go 1.13
 
 require (
-	k8s.io/api v0.17.3
-	k8s.io/apimachinery v0.17.3
+	k8s.io/api v0.18.0
+	k8s.io/apimachinery v0.18.0
 	k8s.io/client-go v0.0.0-20200131194156-19522ff28802 // release-14.0
```

Comment:
client-go is still very old?

Reply:
@msau42 the `hostpath` driver update was performed by the steps mentioned earlier in the comment ( `make` ) , thats why it didnt get update. Do we need to explicitly update the dependency ?

Reply:
i think you need to explicitly update it for the lib (top-most go.mod) to v0.18.0 and then the hostpath provisioner will follow.

Reply:
Apparently that isn't enough. Please also run `go get k8s.io/client-go@v0.18.0`.

Reply:
Wait, client-go is also old in the top-level `go.mod`. *That* is why "make" didn't update it. You need to update it there and then also here.

Reply:
Done.